src/ddmd/dmodule.d

```diff
@@ -112,6 +112,20 @@ extern (C++) const(char)* lookForSourceFile(const(char)** path, const(char)* fil
     return null;
 }
 
+void semantic3OnDependencies(Module m)
```

Comment:
It would be nice to have a comment saying what this function does and a rationale for it.